src/pages/cli/restapi/restapi.mdx

```diff
@@ -135,6 +141,56 @@ REST
           |_ Logs (Amazon CloudWatch)
 ```
 
+## Create REST API and restrict specific routes to specific user groups
+If your app uses user-pool groups to manage different user types and would like to restrict access of specific routes to specifc user groups. we can accomplish this by the following flow: 
```

Comment:
Case convention should be standardized with above reference. User Pool Groups vs. user-pool

Reply:
Got it, will replace all user-pool references with User Pool.